AMPscript extensions checker

FiB3 opened the pull request:
The goal of this PR is to help users regarding the issues #13 & #16 by notifying them about the currently incompatible extensions.

lbuesching commented on src/vscode/src/checks.ts:
```diff
@@ -0,0 +1,31 @@
+	const extensionHandlers = {
+		"sergey-agadzhanov.AMPscript": {
+			handler: () => {
+				vscode.window.showInformationMessage(`MCFS [AMPScript] extension is currently not supported by AMPscript Core.`);
```
Let's update this message to convey what not supported means.

"The extension MCFS [AMPScript] is incompatible with AMPscript Core and will prevent AMPscript Core from debugging AMPscript."

lbuesching commented on src/vscode/src/checks.ts:
```diff
@@ -0,0 +1,31 @@
+			}
+		},
+		"FiB.beautyAmp": {
+			handler: () => true
```
Was this to test a positive test case?

FiB3 commented:
Ok, updated the warning message. :)
You are right, the "beautyAmp" was more of an example and test case. I've removed it and replaced it with a comment.

lbuesching commented:
Backward compatibility checks are fine, no changes from this change.
